src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/IssueCreateDialog.kt

```diff
@@ -70,6 +73,16 @@ internal class IssueCreateDialog(
         return this
     }
 
+    private fun openConfigureFieldsDialog() {
+        val configureFields = driver.wait(elementToBeClickable(configColumnField))
+        try {
+            configureFields.click()
+        } catch (e: ElementClickInterceptedException) {
+            popUps.dismissHealthCheckNotifications() // nobody expects Spanish healthchecks!
```

Comment:
😁

Reply:
I suffer from this as a real user. I have told Jira to "don't remind me again" about this tens of times a week. It doesn't take "no" for an answer 😤